### PR TITLE
docs: vectorstores, different updates and fixes

### PR DESCRIPTION
**Pull request by leo-gan**
# docs: vectorstores, different updates and fixes

Multiple updates:
- added/improved descriptions
- fixed header levels
- added headers
- fixed headers

## Who can review?

Community members can review the PR once tests pass. Tag maintainers/contributors who might be interested:

 @dev2049

### CODE REVIEW

File: docs/modules/indexes/vectorstores/examples/annoy.ipynb
Comment (dev2049) [inline docs/modules/indexes/vectorstores/examples/annoy.ipynb:1]: "Annoy is" not "Annoyis"

File: docs/modules/indexes/vectorstores/examples/atlas.ipynb
Comment (dev2049) [inline docs/modules/indexes/vectorstores/examples/atlas.ipynb:1]: Atlas is a platform... (not Atlas a platform)

File: docs/modules/indexes/vectorstores/examples/tair.ipynb
Comment (dev2049) [inline docs/modules/indexes/vectorstores/examples/tair.ipynb:1]: should have a Tair instance... (not should have an Tair instance)

File: docs/modules/indexes/vectorstores/examples/atlas.ipynb
Comment (leo-gan) [reply docs/modules/indexes/vectorstores/examples/atlas.ipynb:1]: Thanks! Also, the "AtlasDB" title is not correct. The platform's name is "Atlas", and the "AtlasDB" is the name of the LangChain class. So I've change title to "Atlas".

File: docs/modules/indexes/vectorstores/examples/tair.ipynb
Comment (leo-gan) [reply docs/modules/indexes/vectorstores/examples/tair.ipynb:1]: [x]

**Pull request merged**